cumulusci/core/config/project_config.py

```diff
@@ -116,6 +116,18 @@ def _load_config(self):
             }
         )
 
+        self._validate_config()
+
+    def _validate_config(self):
+        """Performs validation checks on the configuration"""
+        self._validate_package_api_format()
+
+    def _validate_package_api_format(self):
+        api_version = self.config["project"]["package"]["api_version"]
+        if isinstance(api_version, int):
+            message = "Package API Version must be specified as a string ('45.0') not an int (45)."
```

Comment:
We should probably explicitly validate that it is the format we need (a string) rather than just checking for int as one bad case. I think yaml like `45.0` might actually end up as a float if it's not in quotes. Someone could even accidentally put in a dict or date or something else.

You asked about validating for "45.0" vs "45". Let's decide that based on what actually works when calling the API. I suspect we need the full dotted version but could you check?